src/users/containers/users/users.component.ts

```diff
@@ -1,38 +1,41 @@
 import { Component, OnInit } from '@angular/core';
-//import { FormGroup } from '@angular/forms';
-//import { FormsService } from '../../../app/containers/form-builder/services/form-builder.service';
-//import { ValidationService } from '../../../app/containers/form-builder/services/form-builder-validation.service';
-//import { select, Store } from '@ngrx/store';
-//import * as fromStore from '../../store';
+import { select, Store } from '@ngrx/store';
+import * as fromStore from '../../store';
 import { Observable } from 'rxjs';
 import { debug } from 'util';
 
 
 
-/**
- * Bootstraps the Register Components
- */
-
 @Component({
   selector: 'app-prd-users-component',
   templateUrl: './users.component.html',
 })
 export class UsersComponent implements OnInit {
 
+  tableConfig: {}[];
+  tableRows: {}[];
```

Comment:
improve types

Reply:
tableRows will remain as it is for now. I introduced a class for tableConfig (=columnConfig)